portality/view/doaj.py

```diff
@@ -342,7 +342,9 @@ def toc(identifier=None, volume=None, issue=None):
 
     # now render all that information
     return render_template('doaj/toc.html', journal=journal, bibjson=bibjson, future=future_journals, past=past_journals,
-                           toc_issns=journal.bibjson().issns())
+                           toc_issns=journal.bibjson().issns(),
+                           std_datetime_fmt=dates.FMT_DATETIME_STD,
+                           default_timestamp=dates.DEFAULT_TIMESTAMP_VAL, )
```

Comment:
Feels like it would be better to add the date format information to a context variable that's already available in the Jinja templates.

For example `constants` is available in all the Jinja templates, so we could attach all the datetime formats to that module instead, then you don't need to pass them into a template like this.

Reply:
I don't think you need to pass these in now, as you've added `dates` as a context variable for the jinja2 templates, so you can just access these properties directly.